### ISSUE TITLE
Duplicate issues between sbt and modules

### ISSUE THREAD

**Issue by eed3si9n** [sbt/sbt#3589]
I am thinking that we should keep GitHub issues open for both sbt/sbt and submodules open, and just label sbt/sbt one as "Duplicate," and close it only when the module with the fix merges to sbt/sbt.

The main motivation is so we can put the one on sbt/sbt into a milestone, and also so we know when things are fixed from sbt/sbt's perspective. What do you think?

Ref https://github.com/sbt/sbt/issues/3569, #3586

**Comment by dwijnand** [comment]
lol the use of the "Duplicate" label :)

**Comment by dwijnand** [comment]
I like that solution. I'd just suggest a label "upstream" (or similar) as "Duplicate" means duplicate within the same issue tracker. Also that way we can use it for other upstream issues.

**Comment by eed3si9n** [comment]
I suggested Duplicate because it's an obvious signal that discussion related to the bug should not take place (unless of course we also label with "Needs discussion").

**Comment by dwijnand** [comment]
I think sbt issues blocked on upstream (which may be a better label than just `upstream`) could still have _some_ discussion, such as: "Is there any way to workaround this JLine issue in sbt?".

I don't think "Duplicate" is the right label. Btw GitHub added some support for the concept of duplication semi recently: https://help.github.com/articles/about-duplicate-issues-and-pull-requests/.

**Comment by eed3si9n** [comment]
ok. Made a new label for this.

**Comment by eed3si9n** [comment]
We should also use `Duplicate of` incantation, if that gives links.
